Article/Article.js

```diff
@@ -9,31 +8,35 @@ class Article {
       We need to make sure the element being passed to us is a jQuery element.
       To do so, wrap the element in jQuery syntax. 
     */
-    this.element = $();
+    this.element = $(element);
     /* 
       Using our new reference to the element, find the expand button within 
       the element and set a new property on the Article class. 
     */
-    this.expandButton;
+    this.expandButton = this.element.find('.expandButton');
     /* 
       Using our new reference to the expandButton, use jQuery to set the inner
       text on the button. 
     */
-
+   $('.expandButton').text('Click to Expand');
     /* 
       Set a click handler on the expand button (or article element), calling 
       the expandArticle method. 
     */
+   this.element.click(() => this.expandArticle());
   }
 
   expandArticle() {
     /* Using our reference to the article element, add or remove a class */
+      $('.article').toggleClass("article-open");
```

Comment:
This currently adds the `"article-open"` class to _every_ article, so when you click on one button, all of the articles expand.